_raml/apps/products/examples/post_product_find_response.json

```
@@ -0,0 +1,468 @@
+{
+  "results": 3,
+  "page": 1,
+  "resultsPerPage": 3,
+  "items": [
+    {
+      "productId": "54EF2E41-FCA8-E0E5-9323-0A0C05E66C2E",
+      "name": "Customised headphone",
```

I would suggest this changed wording cause these headphone are named this way in another example.
```suggestion
      "name": "Boom - Customised headphone",
```